src/admin/server.ts

```diff
@@ -44,10 +56,10 @@ export function getServer(contextFactory: ContextFactory): ApolloServer {
  * Create and start the apollo server. Required to await server.start()
  * before applying middleware per apollo-server 3 migration.
```

Comment:
Should we update this comment, it still mentions apollo 3?

Reply:
👍